### PR TITLE
Xpanse - Generic Playbook

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/36952

## Description
Adding a new generic playbook for Xpanse.
Changing "Expanse Load-Create List" - adding playbook input.
Adding 2 fields to Common.
Changing Expanse layout.
Creating a new mapper, classifier and incident type for Expanse Generic.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yaron-libman** [comment]
@ShirleyDenkberg Please work with Shannon on this one.

File: Packs/ExpanseV2/Scripts/ExpanseRefreshIssueAssets/ExpanseRefreshIssueAssets.py [hunk 1]
```diff
@@ -7,11 +7,33 @@
```
Comment (David-BMS) [inline Packs/ExpanseV2/Scripts/ExpanseRefreshIssueAssets/ExpanseRefreshIssueAssets.py:7]: @iyeshaya Please review and add UT
@tomer-pan FYI

**Review by David-BMS — COMMENTED** [review]
@tomer-pan You can update the from version to be 6.0.0 for all relevant types

File: Packs/ExpanseV2/Scripts/ExpanseRefreshIssueAssets/ExpanseRefreshIssueAssets.py (hunk 1 above)
Comment (tomer-pan) [reply Packs/ExpanseV2/Scripts/ExpanseRefreshIssueAssets/ExpanseRefreshIssueAssets.py:7]: @iyeshaya this is not necessary anymore, I'm removing the changes I have made to the script.

**Review by David-BMS — CHANGES_REQUESTED** [review]

**Review by David-BMS — APPROVED** [review]

**Comment by tomer-pan** [comment]
@shannon-holland @ShirleyDenkberg Do we have a time estimate for the review?

**Comment by shannon-holland** [comment]
@David-BMS Doc review completed.

**Pull request merged**